website/docs/cloud-docs/api-docs/run.mdx

```diff
@@ -100,7 +100,7 @@ The run operation specifies the Terraform execution mode. You can reference the
 
 ### Run Sources
 
-You can use the following sources as query parameters in [workspace runs list](/terraform/cloud-docs/api-docs/run#list-runs-in-a-workspace) and [organization runs list](/terraform/cloud-docs/api-docs/run#list-runs-in-a-workspace).
+You can use the following sources as query parameters in [workspace](/terraform/cloud-docs/api-docs/run#list-runs-in-a-workspace) and [organization](/terraform/cloud-docs/api-docs/run#list-runs-in-a-organization) runs list.
```

Comment:
Wouldn't it be "runs lists"? "...parameters in the workspace and organization runs lists?" That sounds weird.

Reply:
Oh it should be! I did it right above but the typo got me here 😂 

```suggestion
You can use the following sources as query parameters in [workspace](/terraform/cloud-docs/api-docs/run#list-runs-in-a-workspace) and [organization](/terraform/cloud-docs/api-docs/run#list-runs-in-a-organization) runs lists.
```